docs/learn/overview/how-it-works/overview.md

```diff
@@ -1 +1,122 @@
-# How Lit Protocol Works
\ No newline at end of file
+---
+
+slug: /learn/overview/how-it-works
+
+---
+
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# How Lit Protocol Works
+
+Building upon the foundational concepts introduced in the [What is Lit Protocol](../what-is-lit) page, this section delves deeper into the technical architecture and components that make Lit Protocol function under the hood.
+
+Lit Protocol combines advanced cryptography, confidential hardware, and peer-to-peer networking to provide a decentralized platform for key management and secure computation. Below, we explore the core components and mechanisms that enable these capabilities.
+
+## The Lit Nodes
```

Comment:
Maybe we add the list of node operators here?:
https://developer.litprotocol.com/resources/how-it-works#node-operators

Reply:
Nevermind, didn't read far enough